src/node/isometric_positionable.cpp

```diff
@@ -53,6 +59,23 @@ void IsometricPositionable::_enter_tree() {
         ISOMETRIC_SERVER->space_attach_isometric_element(world, self);
     }
     update_position();
+
+#ifdef TOOLS_ENABLED
+    if (Engine::get_singleton()->is_editor_hint() && !Object::cast_to<IsometricMap>(this)) {
```

Comment:
You don't have to test for the class here. You can just use the `is_container `property